packages/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import './accordion/index.test';
```

Comment:
You might not have rebased - this has been moved to **packages/blaze-elements**.